### PR TITLE
Temp vignette (new tv2 branch)

### PR DESCRIPTION
**Pull request by leeyap**
Supersedes #494 - all checks were failing after resolving conflicts. New PR to merge vignette, all changes from previous should have been addressed.

### CODE REVIEW

**Comment by bpbond** [comment]
@leeyap The R builds are erroring — reason seems straightforward if you look at the details. FYI this should also occur if you build on your local system.

File: vignettes/GMAT-GMST.Rmd [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+However, Hector does not produce GMST as an output at this time. Hector does measure the global mean air temperature (GMAT), which we can use to back-calculate GMST. GMAT is the air temperature measured 2 meters above the ground and is what is presented in a daily weather report.^[Taylor, Jessica. "Analyzing Surface Air Temperatures by Latitude." NASA, 2018. https://mynasadata.larc.nasa.gov/mini-lesson/analyzing-surface-air-temperatures-latitude-student-activity]
```
Comment (bpbond) [inline vignettes/GMAT-GMST.Rmd:22]: "Hector does output" or "Hector does calculate"

File: vignettes/GMAT-GMST.Rmd [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+# Rename Hector's variable for GMAT
+global_air_temp <- subset(hector_output, variable == "Tgav")
+global_air_temp <- subset(global_air_temp, select = c(variable, year, value))
+global_air_temp$variable <- "GMAT"
+
+```
+
+## Plot results
+
+The following plot shows GMST and GMAT over time. We see that in about 2000, GMAT begins to increase more quickly than GMST. By 2300, there is a significant gap between the two; GMAT is higher than GMST as GMAT is responding more quickly to anthropogenic forcing. This makes sense as GMST accounts for heating over the oceans which warm more slowly than the atmosphere.
```
Comment (bpbond) [inline vignettes/GMAT-GMST.Rmd:85]: Perhaps: "...GMST accounts for heating of the ocean surface, which warms more slowly than the atmosphere."

**Review by bpbond — APPROVED** [review]
Looks good. Approving pending inclusion of the comment discussed [here](https://github.com/JGCRI/hector/pull/494#discussion_r694042488).

**Comment by leeyap** [comment]
@bpbond All checks passed. Ready to merge if you approve!

**Comment by bpbond** [comment]
Woot woot! Nice job Leeya — thank you.

**Pull request merged**